### ISSUE TITLE
EXP: Better Sweeps on LINCS (`lincs_rdkit_hparam`)

### ISSUE THREAD

**Issue by siboehm** [theislab/chemCPA#69]
I'm not happy with the results of the large sweep that we ran on LINCS. Mainly:
- We should fix the latent space dimension to 64 (or 32?). This also restricts the number of parameters for the Vanilla model.
- We should increase the contribution of the adversarial loss to the AE loss & make the model calculating the disentanglement loss nonlinear. There is no good reason that I can see for using a linear disentanglement loss, it just makes the scores look artificially good.

I'd do it like this: 
1. Run a large sweep (~30 different HPs) for a single embedding (I'd lean towards RDKIT maybe, or Seq2Seq. We could also do GROVER).
2. Use the top performing hyper parameters to train a few models (with different seeds) for each of the other embeddings.
3. Use the results to compare the embeddings in terms of LINCS r2_score (this plot goes into the paper).
4. Use the best performing models for each embedding as the checkpoint for later finetuning on Trapnell.

I think it's important to get this right before we design too many other experiments. 
If it turns out that the transfer learning doesn't help for improving Trapnell scores, then there's no use in implementing #62 for example.
We can then still runs the other experiments like #67 and hope to see improvements there.
If that doesn't work either, we can check whether the model is at least useful for predicting the effects of drugs that it hasn't seen.
That's just for covering our bases in the worst case, I think with some tweaking the Transfer learning will work.
